recipes/flatten_json/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "flatten_json" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 09f34b1f098b97ef2daaa050e68c39c78f8a58515a4e16093487cd5e8f2946a9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - flatten_json
+
+about:
+  home: https://github.com/amirziai/flatten
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Flatten JSON objects'
+
+  description: |
+    Flattens JSON objects in Python. flatten_json flattens the hierarchy in
+    your object which can be useful if you want to force your objects into a table.
+  doc_url:
```

Comment:
Can you either add links here or remove empty entries.

It might make sense for `home`, `doc_url` and `dev_url` to all be `https://github.com/amirziai/flatten`.